portfolio/src/main/webapp/index.html

```diff
@@ -4,13 +4,17 @@
   <meta charset="UTF-8">
   <title>My Portfolio</title>
   <link rel="stylesheet" href="style.css">
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAO0p7PnEgRuKtf1ctUYaVl06AFO9oWBKw"></script>
 </head>
-<body>
+<body onload="createMap()">
```

Comment:
small note, but inline event handlers are discouraged: https://drive.google.com/file/d/0B-yWrS8S9kyqYm1lblRsaTFsQnM/preview

Reply:
Noted. Will change in the future.